pom.xml

```diff
@@ -42,12 +42,15 @@
         </developer>
     </developers>
     <scm>
-        <connection>scm:git:git://github.com/jenkinsci/libvirt-slave-plugin.git</connection>
-        <developerConnection>scm:git:git@github.com:jenkinsci/libvirt-slave-plugin.git</developerConnection>
-        <url>https://github.com/jenkinsci/libvirt-slave-plugin</url>
-        <tag>HEAD</tag>
+        <connection>scm:git:git://github.com/${gitHubRepo}.git</connection>
+        <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+        <url>https://github.com/${gitHubRepo}</url>
+        <tag>${scmTag}</tag>
     </scm>
     <properties>
+        <revision>1.9.3</revision>
+        <changelist>-SNAPSHOT</changelist>
```

Comment:
Should be `<changelist>999999-SNAPSHOT</changelist>` and remove the revision clause.

Reply:
Okay, have created #29